### PR TITLE
allow users to set default values for using fakeseamconnect

### PR DESCRIPTION
**Pull request by kainpets**
Needed for https://github.com/seamapi/api-docs/issues/228

### CODE REVIEW

**Comment by kainpets** [comment]
Screenshot 2024-01-26 at 18 23 47

File: cli.ts [hunk 1]
```diff
@@ -76,6 +77,25 @@ async function cli(args: ParsedArgs) {
     process.exit(0)
   }
 
+  if (args._[0] === "config" && args._[1] === "set" && args._[2] === "server") {
```
Comment (andrii-balitskyi) [inline cli.ts:80]: Probably it's better to make it explicit that it's setting the fake server.
CC @seveibar 

```suggestion
  if (args._[0] === "config" && args._[1] === "set" && args._[2] === "fake-server") {
```

File: cli.ts [hunk 2]
```diff
@@ -76,6 +77,25 @@ async function cli(args: ParsedArgs) {
...
+
+    config.set("server", fakeApiUrl)
+    console.log(`Server URL set to ${fakeApiUrl}`)
+    return
+  }
+
+  if (
+    args._[0] === "config" &&
+    args._[1] === "set" &&
+    args._[2] === "api-key"
```
[the remark this answers is not in the document]
Comment (seveibar) [reply cli.ts:92]: I think he's right on this one, you can set the real api key with this command

File: cli.ts [hunk 3]
```diff
@@ -76,6 +77,22 @@ async function cli(args: ParsedArgs) {
     process.exit(0)
   }
 
+  if (
+    args._[0] === "config" &&
+    args._[1] === "set" &&
+    args._[2] === "fake-server"
+  ) {
+    const randomstring = randomBytes(5).toString("hex")
```
Comment (seveibar) [inline cli.ts:85]: 8**5 is the number of combinations, it's a bit low and could mean a "collision" occurs, let's increase it to 8**8

```suggestion
    const randomstring = randomBytes(8).toString("hex")
```

**Review by seveibar — APPROVED** [review]

**Pull request merged**